### PR TITLE
Update the text and data on the volunteer and partner pages.

### PR DESCRIPTION
**Pull request by seanlip**
Still need to write up three forms and add links to them, but this should fix the rest of the copy.

Also I still need to work on the graph on the partnerships page.

I think these are the only two changes left, though!

### CODE REVIEW

File: static/pages/volunteer/VolunteerProfilesService.js [hunk 1]
```diff
@@ -36,6 +36,8 @@ oppiaFoundationWebsite.factory('VolunteerProfilesService', [
         }
         volunteerProfiles.splice(randomIndex, 1);
       }
+
+      return slides;
```
Comment (hoangviet1993) [inline static/pages/volunteer/VolunteerProfilesService.js]: I think this return statement is unreachable?

File: static/pages/partnerships/partnerships.html [hunk 2]
```diff
@@ -235,7 +243,10 @@ <h3 class="md-display-1 partnerships-page-content-title">
         </md-card-content>
       </div>
     </md-card-content>
-    <md-card-content class="oppia-foundation-no-padding
+  </md-card>
+  <md-card layout-fill
```
Comment (hoangviet1993) [inline static/pages/partnerships/partnerships.html]: Just want to clarify your intention here.

Do you want the student image and the student quote to be its own card(independent section) or do you just want to highlight the image and the quote?

When I received Jordan and Rachel mock, I was under the impression that the image + the quote + the rest of the "We are growing..." is one coherent md-card.
Now it looks like the student and the image is a stand-alone md-card, I would have to tweak its padding (to have 80px top and bottom padding)

**Comment by hoangviet1993** [comment]
I think there should be a
```css
footer a:focus {
  color: white;
}
```
added to the stylesheet. When I tab over the link, it has a blue highlight color, which is kinda hard to see
screen shot 2018-09-17 at 9 39 09 am

File: static/pages/partnerships/partnerships.html (hunk 2 above)
Comment (seanlip) [reply static/pages/partnerships/partnerships.html]: Yep, I changed it to make the student+image more of a "call-out". I noticed on the homepage that the green background was used for short small things we wanted to highlight, so thought I would do that here, too.

So yep, intention is for the student image + quote to be its own card. In the older version it was a bit unclear how it was delineated from the section below.

**Comment by seanlip** [comment]
Hi @hoangviet1993 -- not sure if you want me to merge this, but feel free to do so if you're happy with it! Probably makes sense to merge as the footer change can then come later, but I'm happy to do it in this one later tonight if you prefer.

**Comment by hoangviet1993** [comment]
Thanks @seanlip!

**Pull request merged**